examples/upsert-resource-multiple/.fn-config/fn-config.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: config.kubernetes.io/v1alpha1
```

Comment:
I think we have apiVersion for all functions in catalog ..something starting with `fns.kpt.dev` or something.

pl. ensure it is consistent with other functionConfigs.

/cc @mengqiy

Reply:
We are using `fn.kpt.dev/v1alpha1` for functions authored by the kpt team.

Reply:
We will be using List type, so it would be v1